brunch/brunch-config.js

```diff
@@ -12,6 +12,13 @@ module.exports = {
       joinTo: {'app.js': /^app\/templates/}
     }
   },
+  npm: {
+    globals: {
+      $: 'jquery',
+      Marionette: 'backbone.marionette',
+      Backbone: 'backbone.marionette/node_modules/backbone'
```

Comment:
Why use globals instead of explicitly importing the variables? I appreciate `$` can be unavoidable.